scripts/prepare_systemd_network.sh

```diff
@@ -19,3 +24,27 @@ EOF
 mkdir -p "$WORKING_DIR/etc/systemd/resolved.conf.d"
 
 cp "$TEMPLATES_DIR/cluster.local.conf" "$WORKING_DIR/etc/systemd/resolved.conf.d/"
+
+if [ $IS_IN_AZURE == "true" ]; then
+    cp "$TEMPLATES_DIR/vxlan0.netdev" "$OUTPUT_DIR/vxlan0.netdev"
+
+    echo "
+[Match]
+Name=vxlan0
+
+[Network]
+Address=$VXLAN_IP
+" > $OUTPUT_DIR/foo.network
+
+    for ((i=0; i<${#MASTERS[@]}; i++)); do
+        if [ ${MASTERS[i]} = $NODE_HOST_IP ]; then
+            continue
+        fi
+
+        echo "
+[BridgeFDB]
+MACAddress=00:00:00:00:00:00
+Destination=${MASTERS[i]}
+" >> $OUTPUT_DIR/foo.network
```

Comment:
Let's also rename this file to `vxlan0` :)
```suggestion
" > $OUTPUT_DIR/vxlan0.network

    for ((i=0; i<${#MASTERS[@]}; i++)); do
        if [ ${MASTERS[i]} = $NODE_HOST_IP ]; then
            continue
        fi

        echo "
[BridgeFDB]
MACAddress=00:00:00:00:00:00
Destination=${MASTERS[i]}
" >> $OUTPUT_DIR/vxlan0.network
```